src/pybind/mgr/dashboard/plugins/debug.py

```diff
@@ -33,17 +34,38 @@ class Debug(SP, I.CanCherrypy, I.ConfiguresCherryPy):  # pylint: disable=too-man
         )
     ]
 
+    @no_type_check  # https://github.com/python/mypy/issues/7806
+    def _refresh_health_checks(self):
+        debug = self.get_option(self.NAME)
+        if debug:
+            self.mgr.health_checks.update({'DASHBOARD_DEBUG': {
+                'severity': 'warning',
+                'summary': 'Dashboard debugging is enabled',
```

Comment:
```suggestion
                'summary': 'Dashboard debug mode is enabled',
```
This is a minor difference, but currently the debug mode simply displays tracebacks in HTTP error responses. However, the "debugging" word might resemble like read hard GDB/PDB stuff ongoing and slowing down dashboard performance.